source/guides/network/running-a-test-net-node.rst

```diff
@@ -138,7 +140,7 @@ Once the node is running with ``symbol-bootstrap start`` and you have funded its
 
     symbol-bootstrap link
 
-This creates the required :ref:`VrfKeyLinkTransaction <vrf-key-link-transaction>` and submits it to the network. If it succeeds (it might take some seconds, as the transaction needs to be confirmed) your new node is ready to harvest.
+This creates the required :ref:`AccountKeyLink <account-key-link-transaction>` and :ref:`VrfKeyLink <vrf-key-link-transaction>` transactions and announces them to the network. If it succeeds (it might take some seconds, as the transaction needs to be confirmed) your new node is ready to harvest.
 
 .. note:: Without extra parameters, ``symbol-bootstrap link`` tries to send the registration transaction to the local node (running on the other terminal) which will forward it to the rest of the network. If your node is not running at this moment, or it is not an API node, you can provide the URL of a testnet node using ``--url``. Find a `list of nodes here <https://forum.nem.io/t/nem-symbol-0-10-0-release-announcement-testnet-launch/25863>`_.
```

Comment:
There is a new --useKnownRestGateways in link and enrolSupernode commands that use the best known NGL (highest height) node to announce the transactions. We would keep the list up to date depending on the network type. User doesn't need to have a fully synced local node or manually put an URL to finish up the registration

Reply:
Thanks, added!